aoc/src/main/scala/aoc2015/Day02.scala

```diff
@@ -0,0 +1,15 @@
+package aoc2015
+
+import scala.io.Source
+
+object Day02 extends App:
+
+  val lines: List[String] = Source.fromFile("aoc/src/resources/aoc2015/Day02").getLines.toList
```

Comment:
please define type signature on one line and the implementation on the next line(s), ie.

```
val lines: List[String] =
  Source
    .fromFile("aoc/src/resources/aoc2015/Day02")
    .getLines
    .toList
```